how-to/customize-workspace/public/manifest.fin.json

```diff
@@ -75,6 +75,7 @@
 			"store": true,
 			"dock": true,
 			"notifications": true,
+			"sharing": true,

```

Comment:
Can we add this setting to second.manifest.fin.json so it doesn't lose sharing capability

Reply:
it defaults to true, so not strictly necessary, should we default to false instead and add to second manifest ?

Reply:
Done